src/lib/draw/StreetViewMode.svelte

```diff
@@ -15,38 +9,18 @@
 
   const thisMode = "street-view";
 
-  const circleRadiusPixels = 10;
-
   export let changeMode: (m: Mode) => void;
 
-  export function start() {}
-  export function stop() {
-    cursor = null;
+  export function start() {
+    $map.getCanvas().style.cursor = "zoom-in";
   }
-
-  let cursor: Feature<Point> | null = null;
-
-  // Rendering
```

Comment:
Manually drawing points that followed the cursor was always a bit laggy and verbose code; nice to get rid of this